### ISSUE TITLE
Release new version

### ISSUE THREAD

**Issue by d4rky-pl** [sandelius/mina-puma#15]
Hey, 7fd6d16514e9d is really important for us, could you please bump version and release it?

**Comment by sandelius** [comment]
Yeah the thing is, what if you've set the pid-file in the config file. Then the phased-restart will try using the wrong pid (that's my guess at least).

Could you test to do a phased restart without the --pidfile option?

**Comment by d4rky-pl** [comment]
Well, why don't we just add the same conditional (`if [ -e '#{puma_config}' ]`) as in normal restart? Pumactl supports -F argument for config file so it should be done anyway.

**Comment by sandelius** [comment]
Yeah but it was a bug in it that was fixed just a week ago here: https://github.com/puma/puma/issues/694. 

We should bump the version to 0.3 and set puma version as well.

**Comment by d4rky-pl** [comment]
I guess we'll have to wait until puma releases a new version too. I'll create a pull request that fails for now and ping you when it's ready to merge.

**Comment by sandelius** [comment]
Sounds great. thx
